packages/sanity/src/_internal/cli/commands/migration/runMigrationCommand.ts

```diff
@@ -179,6 +179,14 @@ const runMigrationCommand: CliCommandDefinition<CreateFlags> = {
       return
     }
 
+    output.print(
+      `\n${chalk.yellow(
+        chalk.bold(
+          'Note: If configued, migrations will trigger webhooks. You can disable webhooks on https://manage.sanity.io',
```

Comment:
It might be better to tell users about the `sanity manage` CLI command, which will open the current project  in Manage for them. If they manually navigate to https://manage.sanity.io, they'll first need to find their project.

How about this?

> Run ${chalk.cyan('sanity manage')} to open the management interface in your web browser, then go to the API section to turn off webhooks. You can safely turn them back on after completing the migration.

Note I'd argue we should avoid the term "disable". Here's an excerpt from [Apple's style guide](https://support.apple.com/en-gb/guide/applestyleguide/apsg7af4f5d0/web#apdd167d6f70b0a4):

> disable (v.), disabled (adj.)
> Don’t use disable to refer to turning off a feature or deselecting an option; use turn off or deselect.
> 
> You can turn off Airplane Mode using Control Center.
> To hide an object’s caption, deselect the Caption checkbox.
> 
> Don’t use disabled to describe features that are turned off or unavailable; use turned off, unavailable, or inactive.